src/main/java/nextstep/courses/domain/BaseEntity.java

```diff
@@ -1,11 +1,49 @@
 package nextstep.courses.domain;
 
 import java.time.LocalDateTime;
+import java.util.Objects;
 
 public abstract class BaseEntity {
     protected Long id;
```

Comment:
```suggestion
    private Long id;
```
부모 객체를 생성할 경우 부모의 인스턴스 변수도 가능하면 private으로 구현해보는 도전을 해보는 것을 추천
getter 메서드도 있으니 굳이 protected가 아니여도 되지 않을까?